src/Address/GeocodableAddress.php

```diff
@@ -6,17 +6,24 @@
 
 final class GeocodableAddress
 {
-    private $address;
-    private $postalCode;
-    private $cityName;
-    private $countryCode;
-
+    private $_address;
+    private $_postalCode;
+    private $_cityName;
+    private $_countryCode;
```

Comment:
Property visibilities should not affect their names.

http://www.php-fig.org/psr/psr-2/#42-properties

> Property names SHOULD NOT be prefixed with a single underscore to indicate protected or private visibility.

Reply:
You're all right, Symfony are using psr-2